android/Toodle/app/src/main/java/com/mozilla/toodle/rust/QueryResultValueHandler.java

```diff
@@ -0,0 +1,5 @@
+package com.mozilla.toodle.rust;
+
+public interface QueryResultValueHandler {
```

Comment:
These might be clearer if they used scalar/coll/rel/tuple instead of value/list/rows/row.